src/Components/app/NewsApp.js

```diff
@@ -0,0 +1,44 @@
+import Button from '../button/Button';
+import NewsCard from '../newsCard/NewsCard';
+import './newsApp.scss';
+import NewsService from '../../api';
+
+const template = '<div class="app-container"><ul class="buttons-container"><li class="back-btn">Back</li></ul><ul class="news-container"></ul></div>';
+
+export default class NewsApp {
+  constructor() {
+    const mainContainer = document.querySelector('.app');
+    mainContainer.innerHTML = template;
+  }
+
+  setSource(source) {
+    this.source = source;
+    this.render();
+  }
+
+  async render() {
+    const buttonsContainer = document.querySelector('.app .buttons-container');
```

Comment:
What can happen if the document isn't found in the document? Is it worth considering such a variation?

Reply:
done